x/upgrade/legacy_test.go

```diff
@@ -69,12 +69,9 @@ func (s *LegacyUpgradeTestSuite) SetupSuite() {
 		accounts[i] = tmrand.Str(9)
 	}
 
-	tmCfg := testnode.DefaultTendermintConfig()
-	tmCfg.Consensus.TimeoutCommit = 3 * time.Second
-
 	cfg := testnode.DefaultConfig().
 		WithFundedAccounts(accounts...).
-		WithTendermintConfig(tmCfg).
+		WithTendermintConfig(testnode.DefaultTendermintConfig()).
```

Comment:
can this line be deleted entirely given testnode.DefaultConfig() already uses the default tendermint config?

https://github.com/celestiaorg/celestia-app/blob/047c38b5a7048a404dbc864aeb40e9b81b7f591e/test/util/testnode/config.go#L124

```suggestion
```